apps/meteor/client/views/admin/apps/AppSetting.tsx

```diff
@@ -1,41 +1,64 @@
+import { ISettingBase } from '@rocket.chat/core-typings';
 import { useRouteParameter, useTranslation } from '@rocket.chat/ui-contexts';
-import React, { useMemo, useCallback } from 'react';
+import React, { useMemo, useCallback, ReactElement } from 'react';
 
 import MarkdownText from '../../../components/MarkdownText';
 import MemoizedSetting from '../settings/MemoizedSetting';
 
-const useAppTranslation = (appId) => {
+type AppTranslationFunction = {
+	(key: string, ...replaces: unknown[]): string;
+	has: (key: string | undefined) => boolean;
+};
+const useAppTranslation = (appId: string): AppTranslationFunction => {
```

Comment:
`ReturnType<typeof useTranslation>` might be ok too, I guess...

Reply:
almost... typeof has are incompatible :(